enhancements/network/ovn-kubernetes-evpn.md

```diff
@@ -0,0 +1,417 @@
+* Work on a new downstream enhancement that either replaces or follows this one. 
+
+[1]: https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5089
+
+
+## Motivation
+
+The motivation for this feature is aligned to the one described in the upstream
+[enhancement][2].
+
+[2]: https://github.com/ovn-kubernetes/ovn-kubernetes/blob/4568650412f85cdcf7671c74d2e9b3bb67d3b366/docs/okeps/okep-5088-evpn.md#introduction
+
+### User Stories
+
+The user stories for this feature are aligned to the ones described in the
+upstream [enhancement][3].
+
+[3]: https://github.com/ovn-kubernetes/ovn-kubernetes/blob/4568650412f85cdcf7671c74d2e9b3bb67d3b366/docs/okeps/okep-5088-evpn.md#user-storiesuse-cases
+
+### Goals
+
+The goals for this feature are aligned to the ones described in the upstream
+[enhancement][4].
+
+[4]: https://github.com/ovn-kubernetes/ovn-kubernetes/blob/4568650412f85cdcf7671c74d2e9b3bb67d3b366/docs/okeps/okep-5088-evpn.md#goals
+
+### Non-Goals
+
+The non-goals for this feature are aligned to the ones described in the upstream
+[enhancement][5].
+
+[5]: https://github.com/ovn-kubernetes/ovn-kubernetes/blob/4568650412f85cdcf7671c74d2e9b3bb67d3b366/docs/okeps/okep-5088-evpn.md#non-goals
+
+## Proposal
+
+Ths section requires a general understanding of the overall proposal described
+in the upstream[enhancement][6].
+
+[6]: https://github.com/ovn-kubernetes/ovn-kubernetes/blob/4568650412f85cdcf7671c74d2e9b3bb67d3b366/docs/okeps/okep-5088-evpn.md#proposed-solution
+
+The EVPN feature is mainly driven by the already existing OVN-Kubernetes
+RouteAdvertisements CRD that has no proposed changes. The `routeAdvertisements`
+configuration flag in the OVN-Kubernetes CNO configuration will need to be set
+to `Enabled` to be able to use the feature. Additionally, the EVPN feature
+requires FRR-k8s. FRR-k8s is deployed by CNO when the
+`additionalRoutingCapabilities` `providers` includes `FRR` in the operator
+configuration, which is required to enable `routeAdvertisements`.
+
+The EVPN feature is only supported when `routingViaHost` is set to `true`, also
+known as local gateway mode. CNO will perform no validation, OVN-Kubernetes will
+reject invalid configurations in this regard reporting it through the status of
+the appropriate resources.
+
+The EVPN feature impacts FRR-k8s and OVN-Kubernetes specific APIs in the form of
+new and updated CRDs. A new feature gate will be introduced for the EVPN feature
+and CNO will deploy these CRD updates only if the feature gate is enabled,
+making it impossible to use the feature if the feature gate is not enabled.
+Worth noting that FRR-k8s APIs allow the possibility to inject raw FRR
+configuration and that this capability might be used during the development
+process until proper structured APIs are introduced.
+
+Downstream FRR-k8s image consumes the FRR version from a [RHEL 9.6 base
+image][7] which currently provides FRR v8. The EVPN feature requires FRR v9+
```

Comment:
Wondering if FRR v9+ is not available by GA, Is there a contingency plan or we will stuck with TP for some time until former gets green light?

Reply:
Nothing that I know of for now